app/controllers/application_controller.rb

```diff
@@ -717,12 +717,19 @@ def build_not_found_page(status = 404, layout = false)
       layout = 'application' if layout == 'no_ember'
     end
 
-    category_topic_ids = Category.pluck(:topic_id).compact
+    key = "not_found_page_topics"
```

Comment:
Probably the following so that we are consistent with the naming.

```suggestion
    key = "page_not_found_topics"
```

Also, I think the key should be moved into a constant or a public method so that we're not duplicating this key in multiple places of the code base.

Reply:
I think the name **`not_found_page`**`_topics` was consistent with `build_`**`not_found_page`**, but I changed the name.

The key is used in one place right now so I do not believe it makes sense to move it.